bin/cm.js

```diff
@@ -60,8 +60,8 @@ function error(err) {
   process.exit(1)
 }
 
-function run(cmd, args, wd = root) {
-  return child.execFileSync(cmd, args, {cwd: wd, encoding: "utf8", stdio: ["ignore", "pipe", process.stderr]})
+function run(cmd, args, wd = root, { shell = false }) {
```

Comment:
Isn't this missing an `= {}` to avoid making the argument required? Looks like it will error now when called with 2 or 3 arguments.

Reply:
Yes, thanks. Updated. I've now tested with the status command too.